app/server/lib/shutdown.js

```diff
@@ -65,8 +67,8 @@ function runCleanupHandlers() {
 }
 
 /**
- * Internal helper to exit on a signal. It runs the cleanup handlers, and then re-sends the same
- * signal, which will no longer get caught.
+ * Internal helper to exit on a signal. It runs the cleanup handlers, and then
+ * exit propagating the same signal code than the one caught.
```

Comment:
Oops, maybe just add the missing `s` here.

```suggestion
 * exits propagating the same signal code than the one caught.
```